Lib/tkinter/colorchooser.py

```diff
@@ -19,8 +19,9 @@
 # - title: dialog title
 #
 
-from tkinter.commondialog import Dialog
+__all__ = ["Chooser", "askcolor"]
 
+from tkinter.commondialog import Dialog
```

Comment:
PEP 8: Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants.